thorlcr/graph/thgraph.hpp

```diff
@@ -709,6 +713,10 @@ class graph_decl CGraphBase : public CInterface, implements IEclGraphResults, im
 //  virtual void getResult(size32_t & retSize, void * & ret, unsigned id);
 //  virtual void getLinkedResult(unsigned & count, byte * * & ret, unsigned id);
     virtual IEclGraphResults *evaluate(unsigned parentExtractSz, const byte * parentExtract);
+    virtual void addThorDebugProbe(const char *edgeId, IThorDataLink *p);
+    virtual IThorDataLink *queryThorDataProbe(const char *edgeId);
+    virtual void removeThorDataProbe(const char *edgeId);
```

Comment:
Are addThorDebugProbe()/queryThorDataProbe() and removeThorDataProbe() redundant now?
I don't see them used anywhere.